pkg/controllers/nodeclaim/tagging/suite_test.go

```diff
@@ -213,6 +213,8 @@ var _ = Describe("TaggingController", func() {
 				v1.TagNodeClaim:         nodeClaim.Name,
 				v1.EKSClusterNameTagKey: options.FromContext(ctx).ClusterName,
 			}
+			raw, _ := awsEnv.EC2API.Instances.Load(*ec2Instance.InstanceId)
```

Comment:
ec2Instance is no longer a pointer so it needs to be reloaded after the new tag is added because it is not directly altering the object itself. I can put this into one line but thats the reason for the change.